core/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/Decision.java

```diff
@@ -302,6 +309,13 @@ public String label() {
         }
 
         @Override
+        @Nullable
+        public String getExplanation() {
+            // multi decisions have no explanation
+            return null;
```

Comment:
Sure, it would help to catch incorrect usages, as `getExplanation()` should never be called on `Multi`